src/bugreporter/odemis_bugreporter.py

```diff
@@ -469,21 +488,51 @@ def __init__(self, controller):
         # flag is set to False when the backspace is pressed
         self.make_suggestion = True
 
+        conf_dir = os.path.join(os.path.expanduser(u"~"), '.config', 'odemis')
+        if not os.path.exists(conf_dir):
+            # create the directory so that we can store the config files later
+            os.makedirs(conf_dir)
+        self._config_path = os.path.join(conf_dir, CONFIG_FN)
+        self._names_path = os.path.join(conf_dir, NAMES_FN)
+
+        self._topic_id = self._read_config()
+        logging.debug("Will use topic ID %s. Change by editing %s", self._topic_id, self._config_path)
+
         # Load known users, if not available make tsv file
         self.known_users = OrderedDict()
-        self.conf_path = os.path.join(os.path.expanduser(u"~"), '.config', 'odemis', 'bugreporter_users.tsv')
-        conf_dir = os.path.dirname(self.conf_path)
         try:
-            if not os.path.exists(conf_dir):
-                # create the directory so that we can store the config later
-                os.makedirs(conf_dir)
-            with open(self.conf_path) as f:
+            with open(self._names_path) as f:
                 reader = csv.reader(f, delimiter='\t')
                 for name, email in reader:
                     self.known_users[name] = email
         except (OSError, IOError) as ex:  # IOError is more generic than OSError, and only needed on Python 2
             logging.error("Failed to read known users: %s", ex)
 
+    def _read_config(self):
+        """
+        Read configuration from the bugreporter.config file, and return it
+        return:
+            topic_id (int): the topic ID to use to report bug. In the INI file,
+              it can be stored either as a string from TOPIC_IDS, or an int.
+        """
+        config = configparser.ConfigParser()
+        config.read(self._config_path)
+        topic_id = TOPIC_ID_DEFAULT
+        topic_str = config.get('REPORT', 'topic', fallback=str(TOPIC_ID_DEFAULT))
+        try:
+            try:
+                topic_id = TOPIC_IDS[topic_str]
+            except KeyError:  # Not a known department
+                # Maybe it's just an int?
+                val = int(topic_str)
+                if val <= 0:
+                    raise ValueError("Topic ID must be > 0")
+                topic_id = val
```

Comment:
I'm personally not a fan of nested try except loops, it makes it a bit hard to read. I would go for something like:
```python
topic_id = TOPIC_IDS.get(topic_str, None)
if topic_id is None:  # Not a known department
    # Maybe it's just an int?
    val = int(topic_str)
    if val <= 0:
        raise ValueError("Topic ID must be > 0")
    topic_id = val
```

Do whatever you think is most readable!

Reply:
looks good now